keps/sig-node/NNNN-memory-manager-hugepages-verification/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Memory Manager Hugepages Availability Verification
+kep-number: NNNN
+authors:
+  - "@srikalyan"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2024-12-24
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-node/1769-memory-manager"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
```

Comment:
This should be 1.36, because we can't add features retroactively

Reply:
You're right. I'll update this to v1.36 since we can't add features retroactively. Thanks for catching this.